docs/Endringslogg.adoc

```diff
@@ -26,46 +24,48 @@ Generelt:
+* Klassen «Tjenesteresultat (cv:Output)» utgått, fordi CPSV-AP-NO er nå avgrenset til å støtte metadatabeskrivelser av tjenester og hendelser, mens «Tjenesteresultat (cv:Output)» er ment for å representere et konkret stykke tjenesteresultat fra en gitt tjeneste. Bruk <<MuligTjenesteresultat>> til å metadatabeskrive mulige tjenesteresultater fra en tjeneste. 
 
-* Klassen Understøttende opplysning (cv:SupportedValue) utgått, fordi CPSV-AP-NO er nå avgrenset til å støtte metadatabeskrivelser av tjenester og hendelser, mens Understøttende opplysning (cv:SupportedValue) er ment for å representere en konkret verdi av et informasjonsbegrep som finnes i et konkret stykke dokumentasjon oppgitt av brukeren ved utførelsen av en gitt tjeneste. 
+* Klassen «Understøttende opplysning (cv:SupportedValue)» utgått, fordi CPSV-AP-NO er nå avgrenset til å støtte metadatabeskrivelser av tjenester og hendelser, mens «Understøttende opplysning (cv:SupportedValue)» er ment for å representere en konkret verdi av et informasjonsbegrep som finnes i et konkret stykke dokumentasjon oppgitt av brukeren ved utførelsen av en gitt tjeneste. 
 
 Nye klasser: 
 
-* <<PåkrevdDokumentasjon>>: føyet til for å kunne dekke behov for å beskrive påkrevd dokumentasjon (ettersom klassen cv:Evidence i CPSV-AP skal representere et konkret stykke dokumentasjon som brukeren oppgir ved utførelsen av tjenesten, og ikke hva som kreves av dokumentasjon).
+* <<PåkrevdDokumentasjon>>: føyet til for å kunne dekke behov for å beskrive påkrevd dokumentasjon (ettersom klassen «Dokumentasjon (cv:Evidence)» i CPSV-AP skal representere et konkret stykke dokumentasjon som brukeren oppgir ved utførelsen av tjenesten, og ikke hva som kreves av dokumentasjon).
 
-* <<MuligTjenesteresultat>>: føyet til for å kunne dekke behov for å beskrive hva som er mulige tjenesteresultater fra en tjeneste (ettersom klassen cv:Output i CPSV-AP skal representere et konkret stykke tjenesteresultat fra en gitt tjeneste, og ikke hva som er mulige tjenesteresultater).
+* <<MuligTjenesteresultat>>: føyet til for å kunne dekke behov for å beskrive hva som er mulige tjenesteresultater fra en tjeneste (ettersom klassen «Tjenesteresultat (cv:Output)» i CPSV-AP skal representere et konkret stykke tjenesteresultat fra en gitt tjeneste, og ikke hva som er mulige tjenesteresultater).
 
 Endrede klasser: 
 
 * <<Begrensning>>: 
-** Egenskapen «Begrensning – har understøttende dokumentasjon (cv:hasSupportingEvidence)» utgått (som en konsekvens av at Klassen Dokumentasjon (cv:Evidence) utgår). 
+** Egenskapen «Begrensning – har understøttende dokumentasjon (cv:hasSupportingEvidence)» utgått (som en konsekvens av at klassen «Dokumentasjon (cv:Evidence)» utgår). 
```

Comment:
"utgått" endres til "er utgått" ? Hvis du er enig så gjelder endringen for alle de som er nå utgår.